Stream-Two/Unit03-Testing/challenges/challenge1/test_vending_machine.py

```diff
@@ -0,0 +1,22 @@
+import unittest
+from vending_machine import give_change
+
+
+class TestVendingMachine(unittest.TestCase):
+    """
+    Define a class that inherits from unittest which is the unit
+    testing model that ships with Python.
+    """
+
+    def test_return_change(self):
+        """
+        Method name has to start with test_ in order for it to run.
+        The assertEqual method is inherited from unittest.
+        :return:
+        """
+        coins = give_change(17)
+        self.assertEqual(coins, [10, 05, 02], 'wrong change given')
```

Comment:
You shouldn't start number literals with zeros. In this case it works well, but try out what would happen if you tried to use `08` ;)
These are treated as Octal (8-based) numbers, following a long historical tradition. In Python3, this is actually completely disallowed.